webtau-docs/webtau/REST/JSON-schema.md

```diff
@@ -0,0 +1,32 @@
+# Validation
+
+Webtau supports validation of objects against [JSON Schema](https://json-schema.org/).  For example, to validate the
+entire response body use the `complyWithSchema` matcher as follows:
+
+:include-groovy: scenarios/rest/jsonSchema/validateSchema.groovy {bodyOnly: true, title: "Example schema validation"}
+
+The example schemas are:
+
+:include-json: examples/schemas/valid-schema.json {title: "Correct schema for the example above"}
+:include-json: examples/schemas/invalid-schema.json {title: "Incorrect schema for the example above"}
+
+The second example above will fail with the following error message:
+
+```
+invalid schema (examples/scenarios/rest/jsonSchema/validateSchema.groovy)
+> executing HTTP GET http://192.168.1.5:53830/weather
```

Comment:
for docs sake shall we put some human readable host here?

Reply:
Yes, it does, I just copied it straight from Maven output but I'll make it friendlier.